frontend/kesaseteli/employer/src/pages/index.tsx

```diff
@@ -35,6 +37,18 @@ const EmployerIndex: NextPage = () => {
     }
   }, [isLoading, applications, createApplication, router, locale, isError]);
 
+  const { t } = useTranslation();
+  if (errorMessage) {
+    return (
+      <Notification
+        label={`${t(
+          `common:application.step1.form.common_error`
+        )} ${errorMessage}`}
```

Comment:
This will be replaced by toaster or something better, there is separate task for displaying error messages properly.

Reply:
we implemented the "hds notification + toast" in benefit, but i was not sure that you have the same error handling. so it might be moved then to shared and reused. but then localisation should be moved outside and props should be always translated.

Reply:
Yes there is no toaster in out design pics but I have to ask about it. reusing the same component would be wise.